README.md

```diff
@@ -1,3 +1,69 @@
-# Skynet SDK
+# Skynet NodeJS SDK
 
-An SDK for integrating Skynet into Node.js applications
+An SDK for integrating Skynet into Node.js applications.
+
+## Installing
+
+Using `npm`:
+
+```sh
+npm install @nebulous/skynet
+```
+
+Using `yarn`:
+
+```sh
+yarn add @nebulous/skynet
+```
+
+## Examples
+
+### Uploading and downloading
+
+```js
+const skynet = require("@nebulous/skynet");
+
+const fs = require("fs");
+
+(async () => {
+  // upload file
+
+  const src_file = "./testdata/file1.txt";
+
+  const text = fs.readFileSync(src_file);
+  console.log(`Uploading file ${src_file} containing data: '${text}'`);
+
+  const skylink = await skynet.uploadFile(src_file, skynet.DefaultUploadOptions);
+  if (!skylink.includes(skynet.UriSkynetPrefix)) {
+    console.log(`ERROR: invalid skylink returned: ${skylink}`);
+    return;
+  }
```

Comment:
When could this case be fulfilled? Sounds like either the request was successful and we've got the skylink or it failed and it should reject the promise. Resolving the promise should imply that the request succeeded and the skylink is returned.

Reply:
Removing the readme examples.